src/advanced_examples.md

```diff
@@ -61,24 +61,20 @@ on start {
   print(test3[1])
   print(test3[2])
 
-  let test4 = new Array<int64> [ 0, 1, 2, 3 ]
-  test4[0] = 1
+  let test4 = [ 0, 1, 2, 3 ]
+  test4.set(0, 1)
   print(test4[0])
 
-  print("Map literal assignment")
-  const test5 = new Map<bool, int64> {
-    true: 1
-    false: 0
-  }
+  print("HashMap literal assignment")
+  const test5 = newHashMap(true, 1)
```

Comment:
not `new HashMap`? is this the raw opcode?

Reply:
No, that's the name of the function that will eventually underlie the syntactic sugar. It's not a raw opcode since those have to be 8 characters or less and also because we don't have a HashMap definition in the runtimes, it's [just the name of this Alan function in the root scope](https://github.com/alantech/alan/blob/main/std/root.ln#L612-L618).

I hope to be able to replace that with the syntactic sugar version later (perhaps later this week).